apps/scan/frontend/src/app_root.tsx

```diff
@@ -95,6 +96,18 @@ export function AppRoot({
     refetchInterval: POLLING_INTERVAL_FOR_SCANNER_STATUS_MS,
   });
 
+  const [
+    isCastVoteRecordSyncRequiredScreenUp,
+    setIsCastVoteRecordSyncRequiredScreenUp,
+  ] = useState(false);
+  useQueryChangeListener(usbDriveStatusQuery, {
+    onChange: (newUsbDriveStatus) => {
```

Comment:
A note to @carolinemodic or whoever is going to cherry-pick this PR: since this uses the new API for `useQueryChangeListener`, you'll have to either cherry-pick #4271 first or use the old API in the cherry-picked version of this code.

Reply:
Leaning toward using the old API in the cherry-picked version of this code to keep changes to the release branch minimal